.github/workflows/ci.yml

```diff
@@ -160,7 +153,7 @@ jobs:
     needs: [fmt]
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@stable
         with:
           toolchain: nightly
```

Comment:
We can probably update the coverage job to use `cargo-llvm-cov` and stable Rust (like PyO3 does). Possibly a follow-up?

Reply:
Noted, I think it would be best to align with PyO3's practices. I'd like to take another pass at CI once we get the first release of `pyo3-async-runtimes` out the door.